### ISSUE TITLE
Default Hyperlink Input Type is missing on new installs of 12.02+

### ISSUE THREAD

**Issue by ElBafo** [2sic/2sxc#2474]
I'm submitting a ...
[x] bug report

...about
[x] Content Types or data management

Current behavior
When adding a Hyperlink Data Field you are only shown the Library of Files option, no default option is present in the drop down.

Expected behavior
When adding a Hyperlink Data Field you should be able to choose the default OR library input type

Instructions to Reproduce the Problem
- Install the 2sxc v12.x
- Add the content module
- Add a new content type
- Add a new field
- Select the Hyperlink Data Type
(you will notice that no "default (link,page,file..)" option exists

Your environment
- Windows Server 2019
- SQL Server 2019

2sxc version(s): 12.03.00 Pre-release
Browser: [all]
DNN: [ 9.8.1 | 9.9.1 ] 
Language: [ English ]

Anything you would like to add:
I've also tried it with clean localhost installs (DNN 9.9.1 / DNN 9.8.1) as also clean server installs.
I've installed 2sxc v11 and all seems to be working fine.

![2scx-bug](https://user-images.githubusercontent.com/29287122/124783441-bd84b980-df4d-11eb-8849-50524c399d33.png)

**Comment by iJungleboy** [comment]
It seems that all types after the `@All` (general settings) are missing in 12.03

![image](https://user-images.githubusercontent.com/5321186/124940428-a6f46600-e012-11eb-9021-a1a085cf217f.png)

**Comment by iJungleboy** [comment]
I believe I found the cause.

In ca. 12.01 or 12.02 we cleaned up how the DB is initialized - so that new installs don't have obsolete data. It seems that a few input-definitions without configuration were still retrieved from there. We must now decide how to correct this. Options are:

1. Just create configure-content-types for these input-types, even though they don't have any fields to configure
2. Implement feature to also provide input types at the global level which don't have a configuration-type (ATM not supported)
